app/uk/gov/hmrc/helptosavestub/controllers/NSIController.scala

```diff
@@ -107,6 +112,63 @@ object NSIController extends BaseController with Logging {
 
   private val ninoStatusRegex = """AS(\d{3}).*""".r
 
+  def getAccount(correlationId: Option[String],
+                 nino:          Option[String],
+                 version:       Option[String]): Action[AnyContent] = Action {
+    implicit request ⇒
+      validateParams(correlationId, nino, version)
+        .fold(
+          errors ⇒ {
+            logger.warn("[Handle Account Query] invalid params")
+            errors match {
+              case error :: Nil ⇒ BadRequest(Json.toJson(error))
+              case _            ⇒ BadRequest
+            }
+          }, {
+            maybeNino ⇒
```

Comment:
`maybe` prefix usually indicates an Option but `maybeNino` is a plain `NINO` not an `Option[NINO]`

Can't simply swap the `maybeNino` and `nino` names because that would affect the URL parameter name, so maybe rename `maybeNino` to  `validatedNino` since it has been validated by the `validateParams` function?